authors/apps/articles/views.py

```diff
@@ -129,3 +130,58 @@ def destroy(self, request, *args, **kwargs):
         super().destroy(self, request, *args, **kwargs)
 
         return Response({'message': 'The article has been deleted.'})
+
+
+class RatingAPIView(CreateAPIView, RetrieveUpdateDestroyAPIView):
+    permission_classes = (IsAuthenticated,)
+    queryset = ArticleRating.objects.all()
+    serializer_class = RatingSerializer
+    renderer_classes = (RatingJSONRenderer,)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Users can post article ratings
+        """
+        rating = request.data.get('rating', {})
+
+        try:
+            article = Article.objects.get(slug=kwargs['slug'])
+        except Exception:
```

Comment:
although this works, being more explicit would be better. Change this to except the exact Exception.